grpc/proto/grpc_api.proto

```diff
@@ -56,9 +56,10 @@ message ToServer {
 */
 message FromServer {
     oneof FromServerEnum {
-        UpdateWorkload updateWorkload = 1; /// A message containing lists of workloads to be added or deleted.
-        UpdateWorkloadState updateWorkloadState = 2; /// A message containing list of workload execution states.
-        ank_base.Response response = 3; /// A message containing a response to a previous request.
+        ServerHello serverHello = 1; /// A message containing information about the workloads to be added after the agent connects.
+        UpdateWorkload updateWorkload = 2; /// A message containing lists of workloads to be added or deleted.
+        UpdateWorkloadState updateWorkloadState = 3; /// A message containing list of workload execution states.
+        ank_base.Response response = 4; /// A message containing a response to a previous request.
```

Comment:
Let's not reorder the indices of the messages. Please place the server hello at the end with a new index.
We are not yet handling compatibility between versions, but it is not a good practice to change the indexing without a valid reason.

Reply:
The ServerHello makes sense to be first because of the logic behind it. As for the indexes, I can understand the reason and I will put it at the end.